endpoints/currency/tests/integration_test.js

```diff
@@ -45,6 +45,14 @@ describe('currency', () => {
     })
   })
 
+  describe('searching using provider "cb"', () => {
+    it('should return an array of objects containing correct fields', (done) => {
+      const params = helpers.testRequestParams('/currency/cb')
+      const resultHandler = helpers.testRequestHandlerForFields(done, fieldsToCheckFor)
+      request(params, resultHandler)
```

Comment:
Missing mock data for this test.

Here's an example of how to do it: https://github.com/apis-is/apis/blob/1fb161a/endpoints/horses/tests/integration_test.js#L22-L4